lib/hackmode/mem.js

```diff
@@ -161,44 +125,68 @@ class Mem {
     }
   }
 
-  toJSON () {
-    const out = {}
-    for (let sub of this._data.values()) {
-      for (let addr of sub.keys()) {
-        out[addr] = this.get(addr)
-      }
-    }
-    return out
-  }
+  /**
+   * Regenerate _locHashes
+   * (Called after every insert)
+   * we need a way to generate consistent hashes
+   * (that is, work around insertion order)
+   */
+  _genLocHashes () {
```

Comment:
do we still need this? isn't the gossip logic handled in dht-fullsync now?

Reply:
Mem is still storing dhtEntry and dhtMeta by loc so yes its needed, unless we want to remove that?